src/Uno.UI.Composition/Composition/ContainerVisual.skia.cs

```diff
@@ -45,15 +45,30 @@ internal void ResetRenderOrder()
 		IsChildrenRenderOrderDirty = false;
 	}
 
-	internal override void Draw(in DrawingSession session)
+	internal override void Draw(in DrawingSession session, SKMatrix initialTransform)
```

Comment:
I feel like we should make the parameter `in`, as it's the case with `DrawingSession`?